### ISSUE TITLE
[Fatal] Unhandled top-level exception: $exn  Sexplib.Sexp.of_string: incomplete S-expression while in state Parsing_toplevel_whitespace

### ISSUE THREAD

**Issue by Laxsuperrich** [MinaProtocol/mina#6762]
I run into this error after running command below.
coda daemon -peer-list-file ~/peers.txt -block-producer-key ~/keys/my-wallet -block-producer-password "password" -generate-genesis-proof true

```
 error: {
  "exn_name": "Failure",
  "exn":
    "(Failure\n  \"Sexplib.Sexp.of_string: incomplete S-expression while in state Parsing_toplevel_whitespace: \")"
}
2020-11-19 05:20:58 UTC [Fatal] Unhandled top-level exception: $exn
Generating crash report
        exn: {
  "sexp": [
    "Failure",
    "Sexplib.Sexp.of_string: incomplete S-expression while in state Parsing_toplevel_whitespace: "
  ],
  "backtrace": [
    "Raised at file \"src/error.ml\" (inlined), line 9, characters 14-30",
    "Called from file \"src/app/cli/src/coda.ml\", line 551, characters 12-27",
    "Called from file \"src/deferred1.ml\", line 17, characters 40-45",
    "Called from file \"src/job_queue.ml\" (inlined), line 131, characters 2-5",
    "Called from file \"src/job_queue.ml\", line 171, characters 6-47"
  ]
}


  ☠  Coda Daemon crashed.
   The Coda Protocol developers would like to know why!

    Please:
      Open an issue:
        <https://github.com/CodaProtocol/coda/issues/new>

      Briefly describe what you were doing and attach the crash report /home/ubuntu/.coda-config/coda_crash_report_2020-11-19_05-20-58.851072.tar.gz
```

Crash report is garbled and I can't fix it.

Ubuntu 18.04 (at AWS c5.2xlarge)
[coda_crash_report_2020-11-19_06-46-00.695876.tar.gz](https://github.com/MinaProtocol/mina/files/5564918/coda_crash_report_2020-11-19_06-46-00.695876.tar.gz)

**Comment by mrmr1993** [comment]
@Laxsuperrich I haven't been able to open your crash report file. Are you able to open it locally/do you have another copy of the logs?

**Comment by Laxsuperrich** [comment]
Sorry I still cannot open/read the crash-report and don't have any other copies.

But anyhow my situation got solved.
I don't understand the reason but one of the following commands worked.
(I hit both in a row so I cannot say which)
```
rm -rf ~/.mina-config
rm -rf ~/.coda-config
```

**Comment by mrmr1993** [comment]
Thanks for reporting! I have a suspect root-cause (bad genesis proof) and a proposed fix in PR #6776. If that hypothesis is true, deleting your config directory worked because it contained the bad proof file.

I am closing this now that that PR has merged; we can reopen and investigate further if we get more reports or any other info.
